tests/codeception/acceptance/StanfordEventsCest.php

```diff
@@ -330,7 +330,9 @@ protected function createEventNode(AcceptanceTester $I, $external = FALSE, $cont
       'su_event_subheadline' => 'This is a sub-headline',
     ];
     $values = array_merge($values, $content);
-    return $I->createEntity($values);
+    $e = $I->createEntity($values);
+    $I->runDrush('cache:rebuild');
```

Comment:
That is a lot of `drush cr`. Are you sure it is needed?

Reply:
Similar to the response about enabling the modules in the tests. Since its a module test, it doesn't take codeception long to run anyways. 5 - 6 minutes. adding all the cache rebuilds only adds a couple more seconds each time. I believe if we created the entities only in the UI, we wouldn't need all the cache rebuilding.